### PR TITLE
chore(docs): u-files - minor punctuation and grammar fixes

### PR DESCRIPTION
**Pull request by mendaomn**
As discussed here (https://github.com/gatsbyjs/gatsby/issues/18284#issuecomment-543039978), this PR fixes some minor punctuation and grammar issues with the docs files beginning with a **u**.

## Description

With the aid of [Grammarly](https://www.grammarly.com), I fixed some punctuation issues. In addition, I changed some titles and headings in order to respect [Gatby's casing rules](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#format-titles-and-headers)

## Related Issues

https://github.com/gatsbyjs/gatsby/issues/18284

### CODE REVIEW

File: docs/docs/using-fragments.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Using Fragments
+title: Using GraphQL Fragments
```
Comment (marcysutton) [inline docs/docs/using-fragments.md]: We should probably update the filename as well. Here are some instructions: https://www.gatsbyjs.org/contributing/docs-contributions/#docs-renaming-instructions

File: docs/docs/using-graphql-playground.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Using the GraphQL Playground
 
 ## Intro
 
-In this page we want to introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).
+On this page, we want to introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).
```
Comment (marcysutton) [inline docs/docs/using-graphql-playground.md]:
```suggestion
This page will introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).
```
Might as well update this for the Style Guide.

File: docs/docs/using-fragments.md (hunk 1 above)
Comment (mendaomn) [reply docs/docs/using-fragments.md]: You're right!

File: docs/docs/using-graphql-playground.md (hunk 2 above)
Comment (mendaomn) [reply docs/docs/using-graphql-playground.md]: Thank you!

File: docs/docs/using-fragments.md (hunk 1 above)
Comment (mendaomn) [reply docs/docs/using-fragments.md]: Could you give me some feedback on this? Here's my attempt to follow those instructions :) --> https://github.com/gatsbyjs/gatsby/pull/18762/commits/411fe9869b78a828e5b2f402918a2c6dd0549887

**Pull request merged**